run.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python
+
+import sys
+from os.path import join, dirname, realpath
+
+base_path = dirname(realpath(__file__))
+source_path = join(base_path, "src", "main", "python")
+script_path = join(base_path, "src", "main", "scripts", "cbas")
+
+sys.path.insert(0, source_path)
+exec(open(script_path, 'r').read())
```

Comment:
Sorry, my use case is simple and straight forward: Edit a line in source code and run the program. The `run.py` does exactly that and is good enough and really simple. And yes, I need to have the venv active for it to work properly (or install all dependencies in the OS). And in this use case I don't care about a wrong version because I know that this is *bleeding edge* :-)

If somebody would change the paths then they for sure could also adjust `run.py` or the next person who needs it will adjust it. IMHO your shell function magic is much more complex and much more lines of code. If the built code will ever be significantly different from the source then I'll be happy to PR an improvement.

I would suggest to be pluralistic here: [Everybody should be happy in their own way](https://de.wikipedia.org/wiki/Liste_gefl%C3%BCgelter_Worte/J#Jeder_soll_nach_seiner_Fa.C3.A7on_selig_werden.)